jetty-core/jetty-server/src/main/config/modules/threadpool-all-virtual.mod

```diff
@@ -13,7 +13,16 @@ etc/jetty-threadpool-all-virtual.xml
 
 [ini-template]
 # tag::documentation[]
-## Platform threads name prefix.
+## Virtual threads name prefix.
 #jetty.threadPool.namePrefix=vtp<hashCode>
 
+## Maximum number of concurrent virtual threads.
```

Comment:
I don't think "concurrent" is correct or necessary here.  It makes it sound like there could be a lot more virtual threads, but only a limited number concurrently executing (which is a factor of cores and/or native threads).
```suggestion
## Maximum number of live virtual threads.
```